Source/WebCore/page/ContextMenuController.cpp

```diff
@@ -965,6 +973,10 @@ void ContextMenuController::populate()
         if (selectionIsInsideImageOverlay || (linkURL.isEmpty() && mediaURL.isEmpty() && imageURL.isEmpty())) {
             if (!imageURL.isEmpty())
                 appendItem(*separatorItem(), m_contextMenu.get());
+            
+            auto* ownerElement = frame->ownerElement();
+            bool isPDFDocument = ownerElement ? ownerElement->document().isPDFDocument() : false;
```

Comment:
I think this would look nicer:
`bool isPDFDocument = ownerElement && ownerElement->document().isPDFDocument();`